src/api/nodeRpcCall.js

```diff
@@ -1355,6 +1355,51 @@ const getStakingLedger = async (controller) => {
   return await api.query.staking.ledger(controller);
 }
 
+const stakingChill = async (walletAddress, callback) => {
+  const api = await getApi();
+  const injector = await web3FromAddress(walletAddress);
+  const extrinsic = api.tx.staking.chill();
+  extrinsic.signAndSend(walletAddress, { signer: injector.signer }, ({ status, events, dispatchError }) => {
+    let errorData = handleMyDispatchErrors(dispatchError, api)
+    if (status.isInBlock) {
+      // eslint-disable-next-line no-console
+      console.log(`Completed staking.chill at block hash #${status.asInBlock.toString()}`);
+      callback(null, {
+        blockHash: status.asInBlock.toString(),
+        errorData
+      });
+    }
+  }).catch((error) => {
+    // eslint-disable-next-line no-console
+    console.error(':( staking.chill transaction failed', error);
+    callback({ isError: true, details: error.toString() });
+  });
+}
+
+const stakingValidate = async (walletAddress, callback) => {
+  const api = await getApi();
+  const injector = await web3FromAddress(walletAddress);
+  const extrinsic = api.tx.staking.validate({
+    commission: 100000000, // perbill, 10%
```

Comment:
Lets add a step for the user to set commission andallo/disallow nominating

Reply:
Done:

![image](https://github.com/liberland/liberland_frontend/assets/117277751/b68567cb-1301-4e33-a938-fce815e24b04)